code/Forms/FileFormFactory.php

```diff
@@ -400,6 +400,25 @@ protected function getReplaceFileAction($record)
         return $action;
     }
 
+    /**
+     * Get Download file action
+     *
+     * @param File $record
+     * @return FormAction
+     */
+    protected function getDownloadFileAction($record)
+    {
+        // Check if record exists and user has correct permissions
+        if (!$record || !$record->isInDB() || !$record->canEdit()) {
+            return null;
+        }
+
+        $action = FormAction::create('downloadfile', _t(__CLASS__ . '.DOWNLOAD_FILE', 'Download file'))
+            ->setIcon('install');
```

Comment:
```suggestion
            ->setIcon('p-download');
```

I think p-download would be more appropriate here. But maybe check with @silverstripeux to see what they want. They may want to create a brand new icon

Reply:
'p-download' is used primarily as a site tree icon. Looking below at the screenshot you provided the icon looks a little out of place. I'll look into our fonts and see if there's an appropriate alternative and add to admin. For the meantime it's fine and shouldn't be a blocker.

Reply:
'down-circled' could potentially work better. ![image](https://user-images.githubusercontent.com/24258161/65651100-791e7700-e061-11e9-8999-6fa7babe7e7a.png)